rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -57,10 +58,15 @@ public JsonRpcWeb3ServerHandler(Web3 service, List<ModuleDescription> filteredMo
     protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder request) throws Exception {
         ByteBuf responseContent = Unpooled.buffer();
         int responseCode;
+
         try (ByteBufOutputStream os = new ByteBufOutputStream(responseContent);
-             ByteBufInputStream is = new ByteBufInputStream(request.content().retain())){
-            
+             ByteBufInputStream is = new ByteBufInputStream(request.content().retain())) {
+
             responseCode = jsonRpcServer.handleRequest(is, os);
+        } catch (IllegalArgumentException e) {
+            int errorCode = ErrorResolver.JsonError.PARSE_ERROR.code;
```

Comment:
I'd also log the error here

Reply:
`ErrorResolver.JsonError.INVALID_REQUEST` ?

Reply:
Addressed

Reply:
It should be `PARSE_ERROR (-32700)`  which is what `geth` returns